README.md

```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
 
 Contributions to the osu!wiki are now done through GitHub.
 
-A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [here](/wiki/osu!wiki_contribution_guide).
+A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available at [osu!wiki Contribution Guide](/wiki/osu!wiki_contribution_guide).
```

Comment:
This is just my pedantry speaking, but shouldn't it be "...is available at the osu!wiki Contribution Guide"?